include/fmt/color.h

```diff
@@ -503,8 +503,7 @@ void vprint(std::FILE* f, const text_style& ts, const S& format,
             basic_format_args<buffer_context<Char> > args) {
   basic_memory_buffer<Char> buf;
   internal::vformat_to(buf, ts, to_string_view(format), args);
-  buf.push_back(Char(0));
-  internal::fputs(buf.data(), f);
+  std::fwrite(buf.data(), 1, buf.size(), f);
```

Comment:
To handle `wchar_t` correctly on Windows this should be `fputws` if `Char` is `wchar_t` (#1243). This will truncate output on NUL chars but I don't think we should worry about this. Also use `fwrite_fully` for `char` to handle I/O errors.